app/views/targets/index.html.haml

```diff
@@ -0,0 +1,30 @@
+#googleMap
+%script{:src => "http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"}
+:javascript
+  function myMap() {
+  var mapProp= {
+      center:new google.maps.LatLng(-34.906268,-56.186120),
+      zoom:15,
+  };
+  var map=new google.maps.Map(document.getElementById("googleMap"),mapProp);
+  }
+%script{:src => "https://maps.googleapis.com/maps/api/js?key=AIzaSyAiW1JcyQxgKFXNbSLy19mciRsUqSneuE4&callback=myMap"}
+#rectangle-3
```

Comment:
do not use ID's unless you are going to use those as reference on a javascript function.